src/components/dataset-api-doc/api-example.test.js

```diff
@@ -6,11 +6,13 @@ describe('api example', () => {
     const params = [
       {
         name: 'bob',
+        default: 'smith',
```

Comment:
Why is the default 'smith'? This isn't what we use when they don't send this parameter.

Reply:
this is just the test, so it the value shouldn't matter.